enhancements/windows-containers/health-management.md

```diff
@@ -87,7 +87,9 @@ To enable this, WMCO will have these changes in responsibility:
 * WMCO will create and maintain a ConfigMap which provides WICD with the specifications for each Windows service that
   must be created on a Windows instance.
 * WMCO will invoke the `bootstrap` command of WICD when initially adding a Windows instance to the cluster. 
-  This will involve the services ConfigMap in order to start the containerd runtime and kubelet services.
+  This will require the services ConfigMap in order to start bootstrap services like the containerd runtime and kubelet
+  services. WMCO will pass the desired version of the services ConfigMap to use as the source of truth as an argument to
```

Comment:
`WMCO will pass the desired version of the services ConfigMap`
consider changing this to `the current WMCO version will be passed as an argument to WICD 'bootstrap', indicating which services ConfigMap to use`

Reply:
+1 to this